Makefile

```diff
@@ -1,8 +1,8 @@
 Version := $(shell git describe --tags --dirty)
 GitCommit := $(shell git rev-parse HEAD)
 LDFLAGS := "-s -w -X main.Version=$(Version) -X main.GitCommit=$(GitCommit)"
-CONTAINERD_VER := 1.3.2
-CNI_VERSION := v0.8.5
+CONTAINERD_VER := v1.3.4
+CNI_VERSION := v0.8.6
```

Comment:
This doesn't appear to have any bearing on errors.

Reply:
should the DEV docs be updated to install 1.3.4 now?

Reply:
If this passes